src/character.cpp

```diff
@@ -1491,6 +1492,41 @@ bool Character::can_run() const
     return get_stamina() > 0 && !has_effect( effect_winded ) && get_working_leg_count() >= 2;
 }
 
+bool Character::is_bp_armored( const bodypart_id &bp ) const
+{
+    for( const bionic_id &bid : get_bionics() ) {
+        if( find( bid->covered_bodyparts.begin(), bid->covered_bodyparts.end(),
+                  bp.id() ) != bid->covered_bodyparts.end() && bid->has_flag( flag_BIO_CANT_COMPRESS ) ) {
+            return true;
+        }
+    }
+    return false;
+}
+
+double Character::bp_bite_chance( const bodypart_id &bp ) const
```

Comment:
Shouldn't this return a float instead? Is double necessary?

Reply:
because
![image](https://user-images.githubusercontent.com/44523519/101141773-bc09b080-364f-11eb-8dab-5ce96b2c1e4c.png)